app/client/src/ce/pages/Applications/index.tsx

```diff
@@ -98,6 +98,7 @@ import ApplicationCardList from "./ApplicationCardList";
 import { usePackage } from "@appsmith/pages/Applications/helpers";
 import PackageCardList from "@appsmith/pages/Applications/PackageCardList";
 import WorkspaceAction from "@appsmith/pages/Applications/WorkspaceAction";
+import ResourceListLoader from "./ResourceListLoader";
```

Comment:
This should ideally be
```suggestion
import ResourceListLoader from "@appsmith/pages/Applications/ResourceListLoader";
```